backend/src/layer/logic/mealplan_management/meal_plan_manager.rs

```diff
@@ -33,7 +33,7 @@ where
         for parse_canteen in parse_canteens {
             let name = &parse_canteen.name.clone();
             match self.resolver.resolve(parse_canteen, date).await {
-                Ok(_canteen) => debug!("resolved canteen '{name}' with no errors"),
+                Ok(_) => debug!("resolved canteen '{name}' with no errors"),
```

Comment:
afaik `resolve`returns `Result<()>`, so writing `Ok(())` would make clear, that no information is left unused here.

Reply:
I don't get the issue here. Resolve needs matching.